colcon_sanitizer_reports/event_handlers/sanitizer_report.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Open Source Robotics Foundation
+# Licensed under the Apache License, version 2.0
+
+from colcon_core.event.job import JobEnded
+from colcon_core.event_handler import EventHandlerExtensionPoint
+from colcon_core.location import get_log_path
+from colcon_core.plugin_system import satisfies_version
+from colcon_output.event_handler.log import STDOUT_STDERR_LOG_FILENAME
+from colcon_sanitizer_reports.sanitizer_log_parser import SanitizerLogParser
+
+
+class SanitizerReportEventHandler(EventHandlerExtensionPoint):
+    """Generate a report of all Sanitizer ERRORs and WARNINGs."""
+
+    ENABLED_BY_DEFAULT = False
+
+    def __init__(self):
+        super().__init__()
+        satisfies_version(EventHandlerExtensionPoint.EXTENSION_POINT_VERSION, '^1.0')
+        self.enabled = False
```

Comment:
Why is this not _ENABLED_BY_DEFAULT?

Reply:
Agreed. It seems the goal here is to use the constant as the default value for this constructor. See https://github.com/colcon/colcon-output/blob/ad340e7c70e2c72a018dc483c209a6ab8c2db58b/colcon_output/event_handler/console_cohesion.py#L23-L38